src/coreComponents/mesh/generators/InternalMeshGenerator.hpp

```diff
@@ -52,6 +52,21 @@ class InternalMeshGenerator : public MeshGeneratorBase
 
   virtual void generateMesh( DomainPartition & domain ) override;
 
+  /**
+   * @brief import fields from the mesh  on the array accessible via the given wrapper.
+   * @param cellBlockName name of the cell block to copy data from.
+   * @param meshFieldName name of the field in the meshd
+   * @param isMaterialField Indicate if we want to import material or regular fields
+   * @param wrapper Wrapper to access the array
+   */
```

Comment:
```suggestion
```
I do not think you need to doxygen a derived function if it's the exact same doc.